test/loading.jl

```diff
@@ -381,5 +381,10 @@ const uuidT = UUID("a54bd003-d8dc-4161-b186-d5516cd448e9")
     end
 end
 
+@testset "Issue #25719" begin
+    @test Base.root_module(Base, :Test) == Test
```

Comment:
Oddly enough this works on master but is an error here, so I don't know what the deal is. I'll just close this unless anyone has any suggestions.